rfc6962-bis.xml

```diff
@@ -541,9 +541,7 @@ ranges.
         v2(1), (255)
     } Version;
 
-    struct {
-        opaque key_id[HASH_SIZE];
-    } LogID;
+    opaque LogID&lt;1..2^8-1&gt;;
```

Comment:
The OBJECT IDENTIFIER octet and the ASN.1 length octet(s) are stripped from the DER-encoded OID, leaving just the ASN.1 data octets.  Those ASN.1 data octets are what is stored in the opaque field of 1 - 255 octets.
If needed, the DER-encoded OID can be easily reconstructed from just those data octets: the ASN.1 tag octet is always 0x06; and if the length of the opaque field is <=127 octets (which is true for 100% of the OIDs I've even seen!), the ASN.1 length octet will be identical to the opaque field length.  (In other words, you can convert between the two forms by prepending a 0x06 octet or removing the leading 0x06 octet).